lib/utils/constants.dart

```diff
@@ -169,6 +170,7 @@ const kHintPrice = '10.87';
 const kHintHashtag = 'Type in';
 const kHashtagsText = 'Hashtags (optional)';
 const kAddText = 'Add';
+const kNetworkFeeWarnText = 'A network fee of 10% the listed price is required for all transactions that occur on-chain';
```

Comment:
need to add another "of" after the word "10%"

Reply:
@mustafapylons , it's fixed now.